src/docs/concepts-development/sitemap/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Sitemap support in Sharetribe
+slug: sitemap-in-sharetribe
+updated: 2023-10-12
+category: concepts-development
+ingress:
+  This guide describes how the Sharetribe tooling supports creating a
+  sitemap for your marketplace.
+published: true
+---
+
```

Comment:
Useful links for people who read this article:

- https://developers.google.com/search/docs/crawling-indexing/sitemaps/build-sitemap#xml
- https://developers.google.com/search/docs/crawling-indexing/sitemaps/large-sitemaps
- https://developers.google.com/search/docs/crawling-indexing/robots/intro